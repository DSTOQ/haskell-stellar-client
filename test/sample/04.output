TransactionEnvelope
  { _transaction =
      Transaction
        { _sourceAccount =
            PublicKeyEd25519
              { _publicKeyEd25519 =
                  [ 99
                  , 56
                  , 97
                  , 50
                  , 55
                  , 52
                  , 57
                  , 98
                  , 49
                  , 55
                  , 54
                  , 51
                  , 100
                  , 48
                  , 48
                  , 55
                  , 50
                  , 52
                  , 101
                  , 97
                  , 55
                  , 97
                  , 53
                  , 48
                  , 48
                  , 98
                  , 49
                  , 49
                  , 51
                  , 98
                  , 101
                  , 98
                  , 52
                  , 97
                  , 51
                  , 97
                  , 53
                  , 56
                  , 99
                  , 50
                  , 56
                  , 50
                  , 50
                  , 51
                  , 99
                  , 50
                  , 50
                  , 98
                  , 52
                  , 53
                  , 97
                  , 99
                  , 102
                  , 54
                  , 102
                  , 50
                  , 54
                  , 97
                  , 48
                  , 57
                  , 56
                  , 54
                  , 49
                  , 99
                  ]
              }
        , _fee = Fee { _fee = 100 }
        , _seqNum = SequenceNumber { _sequenceNumber = 46373822751834113 }
        , _timeBounds = Nothing
        , _memo = MemoNone
        , _operations =
            [ Operation
                { _sourceAccount = Nothing
                , _body =
                    ChangeTrust
                      ChangeTrustOp
                        { _line =
                            AssetCreditAlphanum
                              AssetCode { _assetCode = "HTKN" }
                              PublicKeyEd25519
                                { _publicKeyEd25519 =
                                    [ 54
                                    , 98
                                    , 98
                                    , 54
                                    , 54
                                    , 98
                                    , 53
                                    , 99
                                    , 52
                                    , 52
                                    , 97
                                    , 52
                                    , 48
                                    , 102
                                    , 100
                                    , 56
                                    , 50
                                    , 99
                                    , 52
                                    , 52
                                    , 97
                                    , 101
                                    , 54
                                    , 98
                                    , 54
                                    , 101
                                    , 98
                                    , 53
                                    , 101
                                    , 99
                                    , 99
                                    , 49
                                    , 99
                                    , 98
                                    , 55
                                    , 56
                                    , 99
                                    , 100
                                    , 99
                                    , 48
                                    , 54
                                    , 97
                                    , 54
                                    , 49
                                    , 98
                                    , 54
                                    , 102
                                    , 56
                                    , 51
                                    , 51
                                    , 57
                                    , 101
                                    , 99
                                    , 97
                                    , 98
                                    , 98
                                    , 55
                                    , 49
                                    , 54
                                    , 100
                                    , 54
                                    , 100
                                    , 100
                                    , 56
                                    ]
                                }
                        , _limit = Just 9223372036854775807
                        }
                }
            ]
        }
  , _signatures =
      [ DecoratedSignature
          { _hint = SignatureHint [ 54 , 97 , 48 , 57 , 56 , 54 , 49 , 99 ]
          , _signature =
              Signature
                { _signature =
                    "[99,48,57,97,55,99,51,50,52,55,55,50,54,48,97,102,55,54,100,56,53,56,53,101,51,98,54,49,49,49,100,52,56,99,52,50,101,99,100,55,101,102,101,99,49,52,55,99,98,97,97,98,49,102,97,51,48,54,102,102,54,51,52,98,100,53,57,102,100,49,100,100,50,57,57,56,54,50,56,102,57,52,52,56,48,102,57,100,100,99,102,51,98,55,102,97,56,56,50,51,52,50,50,57,52,51,49,99,100,51,56,50,102,49,102,101,99,51,56,48,99,50,101,99,97,53,48,51]"
                }
          }
      ]
  }