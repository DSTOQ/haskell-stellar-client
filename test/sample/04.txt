AAAAAMiidJsXY9AHJOp6UAsRO+tKOljCgiPCK0Ws9vJqCYYcAAAAZACkwL8AAAABAAAAAAAAAAAAAAABAAAAAAAAAAYAAAABSFRLTgAAAABrtmtcRKQP2CxErmtutezBy3jNwGphtvgznsq7cW1t2H//////////AAAAAAAAAAFqCYYcAAAAQMCafDJHcmCvdthYXjthEdSMQuzX7+wUfLqrH6MG/2NL1Z/R3SmYYo+USA+d3PO3+ogjQilDHNOC8f7DgMLspQM=
