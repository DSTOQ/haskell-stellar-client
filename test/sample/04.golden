TransactionEnvelope
  { transaction =
      Transaction
        { sourceAccount =
            PublicKeyEd25519
              { unPublicKeyEd25519 =
                  90749603724978580283934445940931038574922316733825256476224977185719745873436
              }
        , fee = Fee { unFee = 100 }
        , seqNum = SequenceNumber { unSequenceNumber = 46373822751834113 }
        , timeBounds = Nothing
        , memo = MemoNone
        , operations =
            VarLen
              { unVarLen =
                  [ Operation
                      { sourceAccount = Nothing
                      , body =
                          ChangeTrust
                            ChangeTrustOp
                              { line =
                                  AssetCreditAlphanum4
                                    AssetCode4 { unAssetCode4 = 1213483854 }
                                    PublicKeyEd25519
                                      { unPublicKeyEd25519 =
                                          48719781938602096453436114010251764830087310142640098156249894279754447416792
                                      }
                              , limit = Just 9223372036854775807
                              }
                      }
                  ]
              }
        }
  , signatures =
      VarLen
        { unVarLen =
            [ DecoratedSignature
                { hint = SignatureHint { unSignatureHint = 1779009052 }
                , signature =
                    Signature
                      { unSignature =
                          VarLen
                            { unVarLen =
                                "\192\154|2Gr`\175v\216X^;a\DC1\212\140B\236\215\239\236\DC4|\186\171\US\163\ACK\255cK\213\159\209\221)\152b\143\148H\SI\157\220\243\183\250\136#B)C\FS\211\130\241\254\195\128\194\236\165\ETX"
                            }
                      }
                }
            ]
        }
  }